spec/models/user_spec.rb

```diff
@@ -168,4 +168,23 @@
       end
     end
   end
+
+  describe '#inventories' do
+    let(:user) { FactoryGirl.create(:user, first_name: 'John', last_name: 'Doe') }
+    context 'with both a first and last name' do
```

Comment:
The context here is wrong, but the test here is correct.

Main things to note:  we could probably move the creation of those entities into two separate contexts; one test covering when the user is the owner, and one when the user isn't.  But this satisfies the requirements.

This specific spec be addressed when we go back to spruce up the code base.